<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebMarkTestClass</class>
 <widget class="QMainWindow" name="WebMarkTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebMarkTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>html</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QWebEngineView" name="webEngineView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>picture</string>
         </attribute>
         <widget class="QLabel" name="img_lb">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>1131</width>
            <height>721</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="openHtml_btn">
          <property name="text">
           <string>打开html</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="path_lEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_btn">
          <property name="text">
           <string>加载</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendData_btn">
          <property name="text">
           <string>发送测试</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WebMarkTest.qrc"/>
 </resources>
 <connections/>
</ui>
